README.md

```diff
@@ -27,46 +27,56 @@ Requirements
-  - `gitlab_host`  
+  - `gitlab_runner_skip_registration`
+  Skip gitlab-runner registration after installation. Default value is `false`
+  - `gitlab_host`
   Docker gitlab server. Default value is `git.epam.com`
-  - `gitlab_runner_tags`  
+  - `gitlab_runner_tags`
   The tags associated with the Runner. Should be comma delimited. Default value is `delegated`
-  - `gitlab_runner_untagged_builds_run`  
+  - `gitlab_runner_untagged_builds_run`
   Config that prevents it from picking untagged jobs. Default value is `false`
-  - `gitlab_runner_lock_to_project`  
+  - `gitlab_runner_lock_to_project`
   Config that lock the Runner to current project. Default value is `false`
-  - `gitlab_runner_executor`  
+  - `gitlab_runner_executor`
   Runner executor. Default value is `shell`
-  - `gitlab_runner_extra_options`  
-  Extra option for runner registration process. Default value is `undefined`  
-  - `gitlab_runner_limit`  
+  - `gitlab_runner_extra_options`
+  Extra option for runner registration process. Default value is `undefined`
+  - `gitlab_runner_limit`
   Config that Limit how many jobs can be handled concurrently by this token. `0` simply means don't limit. Default value is `1`
-  - `gitlab_runner_concurrent`  
+  - `gitlab_runner_concurrent`
   Limits how many jobs globally can be run concurrently.
-  The most upper limit of jobs using all defined runners. 
+  The most upper limit of jobs using all defined runners.
   0 does not mean unlimited. Default value is `ansible_processor_vcpus`
-  - `gitlab_runner_request_concurrency`  
+  - `gitlab_runner_request_concurrency`
   Limit number of concurrent requests for new jobs from GitLab. Default value is `1`
-  - `gitlab_runner_env_vars`  
+  - `gitlab_runner_env_vars`
   Append or overwrite environment variables. Default value is `["ENV=value", "LC_ALL=en_US.UTF-8"]`
-  - `packages_additional`  
+  - `packages_additional`
   Install additional packages for all installs. Default value is `[]`
-  - `gitlab_runner_gpg`  
-  GPG key for Debian. Default value is `https://packages.gitlab.com/gpg.key`  
+  - `gitlab_runner_gpg`
+  GPG key for Debian. Default value is `https://packages.gitlab.com/gpg.key`
+
+- advanced configuration
+  - `gitlab_runner_config`
+  Dictionary used for advanced configs:
+    - `params`
+    Environment variables used during the registration process
+    - `global_values`
+    Dictionary with *key:value* used to add/change ***non string values*** in the file "config.toml"
+    - `global_strings`
+    Dictionary with *key:value* used to add/change ***string values*** in the file "config.toml"
 
 ## Some examples of the installing current role
 
 With [playlabs](https://yourlabs.io/oss/playlabs) you can install this role with just one command, ie:
 
-    playlabs install lean_delivery.gitlab_runner @localhost gitlab_ci_token=yourcommand gitlab_host=yourlabs.io gitlab_runner_limit=4 gitlab_version=11.6 
```

Comment:
Should be reverted empty space

Reply:
> Should be reverted empty space

The empty space at the end of the lines? What's the purpose?

Reply:
README formatting ![image](https://user-images.githubusercontent.com/9392033/66826545-593ded00-ef55-11e9-917b-023b306997cd.png)

Reply:
got it, I've setup my editor to trim automatically the whitespace. I don't think I'm the only one that does it, what about overriding the whitespaces with a `<br>`?

Reply:
> got it, I've setup my editor to trim automatically the whitespace. I don't think I'm the only one that does it, what about overriding the whitespaces with a `<br>`?

You could add README.md to ignore automatically trim spaces

Reply:
> > got it, I've setup my editor to trim automatically the whitespace. I don't think I'm the only one that does it, what about overriding the whitespaces with a `<br>`?
> 
> You could add README.md to ignore automatically trim spaces

Thanks didn't know how to do it. Done and I'm going to push the changes

Reply:
done